{"author":{"id":"f3e27cc0429217a3cfc9d2fedfec965b6caf30730b7ac739aa7509c4654af6cd"},"ops":[{"type":6,"timestamp":1588497536,"nonce":"KMdSVE9Mw/sLXzgV+URoQyTjQ80=","metadata":{"github-id":"MDE1OlVzZXJDb250ZW50RWRpdElzc3VlRWRpdDozNzczODIyNDY="},"target":"dfe6207fdd70ad7558f3f9627b43693b8c1fcc7c1110931b4e28846cf57aa324","message":"git-bug version: 0.7\ngitlab server: GitLab Community Edition 12.10.2 \n\nI get a bug (or feature) with the `bridge pull` command. `git-bug` do not crash, but stop importing after an error.\n\n```bash\nimport error at id 975: label event creation: removed label: empty\nimported 643 issues and 13 identities... \n```\n\nThen...\n- all issues after the error are missing.\n- Many issue closed are open in git-bug.\n```\nGitLab: Open 76    Closed 697    All 773\ngit-bug: open 640    closed 3\n```\n\nIf I run again the command, issues are added again so I have them twice !\n\n```bash \nimport error at id 975: label event creation: removed label: empty\nimported 643 issues and 0 identities with git.duniter.org bridge\n```\nThen:\n```\nGitLab: Open 76    Closed 697    All 773\ngit-bug: open 1280     closed 6 \n```\n\nWas so close to heaven ! ;)","files":null}]}